build.gradle.kts

```diff
@@ -3,8 +3,8 @@ buildscript {
 }
 
 plugins {
-  kotlin("jvm") version "1.4.30" apply false
-  id("org.jetbrains.dokka") version "0.10.1" apply false
+  kotlin("jvm") version "1.5.0" apply false
```

Comment:
1.5.10 was just cut today, may as well update to that instead?
https://github.com/JetBrains/kotlin/releases/tag/v1.5.10